packages/sdks/react-sdk/src/hooks/useSession.ts

```diff
@@ -5,19 +5,20 @@ const useSession = () => {
   const {
     session,
     isSessionLoading,
+    isOidcFinishLogin,
     fetchSession,
     isSessionFetched,
     isAuthenticated,
   } = useContext();
 
   // when session should be received, we want the return value of "isSessionLoading" to be true starting from the first call
   // (and not only when receiving an update from the context)
-  const isLoading = useRef(isSessionLoading);
+  const isLoading = useRef(isSessionLoading || isOidcFinishLogin);
```

Comment:
Maybe we should call it `isOidcLoading`?
`finish login` sounds to me like the login is already finished

Reply:
good point, adjusted 👍